frontend/libs/health-department/feature-case-detail/src/lib/diary-entries-list/diary-entries-list.component.ts

```diff
@@ -0,0 +1,98 @@
+import { Component, OnInit } from '@angular/core';
+import { CaseDto, CaseEntityService, TrackedCaseDiaryEntryDto } from '@qro/health-department/domain';
+import * as _ from 'lodash';
+import { Dictionary } from '@ngrx/entity';
+import { DiaryListItemModel } from '../diary-entries-list-item/diary-entries-list-item.component';
+import { ActivatedRoute } from '@angular/router';
+import { map, shareReplay, switchMap, tap } from 'rxjs/operators';
+import { Observable } from 'rxjs';
+import { ApiService } from '@qro/shared/util-data-access';
+
+@Component({
+  selector: 'qro-diary-entries-list',
+  templateUrl: './diary-entries-list.component.html',
+  styleUrls: ['./diary-entries-list.component.scss'],
+})
+export class DiaryEntriesListComponent implements OnInit {
+  listItems$: Observable<DiaryListItemModel[]>;
+  private caseId: string;
+
+  private readonly MORNING = 'morning';
+  private readonly EVENING = 'evening';
+
+  constructor(
+    private route: ActivatedRoute,
+    private entityService: CaseEntityService,
+    private apiService: ApiService
+  ) {}
+
+  ngOnInit() {
+    this.listItems$ = this.route.parent.paramMap.pipe(
+      tap((params) => (this.caseId = params.get('id'))),
+      switchMap((params) => this.entityService.loadOneFromStore(params.get('id'))),
+      switchMap((caseDto: CaseDto) => this.apiService.getApiCall(caseDto, 'diary')),
+      map((response) => {
+        return this.mapToListItems(this.mapToTrackedCaseDiaryEntryDtoList(response));
+      }),
+      shareReplay(1)
+    );
+  }
+
+  private mapToTrackedCaseDiaryEntryDtoList(diaryEntriesList: any): TrackedCaseDiaryEntryDto[] {
+    return diaryEntriesList?._embedded?.trackedCaseDiaryEntrySummaryList?.map((entry) => {
+      return {
+        bodyTemperature: entry.bodyTemperature,
+        timeOfDay: entry.slot.timeOfDay,
+        date: entry.slot.date,
+        symptoms: entry.symptoms.map((symtom) => symtom.name),
+        contacts: entry.contacts.map((contact) => contact.firstName + ' ' + contact.lastName),
```

Comment:
Es wird Kontakte geben, bei denen weder firstName noch lastName gepflegt ist. In dem Fall sollte hier vielleicht was anderes angezeigt werden, z. B. "Anonymer Kontakt"? -> Im Zweifel mit Jochen oder Patrick klären